### DISCUSSION TITLE
convert svg to png lose color

### DISCUSSION THREAD

**Discussion by doudouodong** [libvips/libvips#4067] [General]
## Questions, enhancements, tips, etc.

Please use libvips discussions for other topics.

https://github.com/libvips/libvips/discussions

## Bug report
![123](https://github.com/user-attachments/assets/a9cdaddf-dc68-4f54-995a-6bcf707b337d)

vips::VImage image = vips::VImage::new_from_file("123.svg", vips::VImage::option ()->set ("access", VIPS_ACCESS_SEQUENTIAL));
image.write_to_file("123.png");  

svg to png lose color

**Describe the bug**

**To Reproduce**
1. Use Image '...'
2. Use Configuration '....'
3. See error

**Expected behavior**

**Actual behavior**
A clear and concise description of what actually happen.

**Screenshots**

**Environment**
(please complete the following information)
 - OS: [e.g. Ubuntu 16.04, MacOS 10.13]
 - Vips: [e.g. 8.8, 8.9, etc]

**Additional context**

**Comment by lovell** [comment]
```xml
<style>:root{--color-0:#6A00BC}</style>
```
Please see https://gitlab.gnome.org/GNOME/librsvg/-/issues/459
